readme.md

```diff
@@ -62,4 +62,4 @@ Finally, run `npm start` to run Gulp. Your finished site will be created in a fo
 http://localhost:8000
 ```
 
-To create compressed, production-ready assets, run `npm run build`.
+To create compressed, production-ready assets, run `npm run build -- --production`.
```

Comment:
Hey Colin, it is not a typo. This is the way to pass arguments to npm scripts 
https://docs.npmjs.com/cli/run-script

```
As of npm@2.0.0, you can use custom arguments when executing scripts. The special option -- is used by getopt to delimit the end of the options. npm will pass all the arguments after the -- directly to your script:

npm run test -- --grep="pattern"
The arguments will only be passed to the script specified after npm run and not to any pre or post script.
```